musig/src/tests.rs

```diff
@@ -0,0 +1,342 @@
+#[cfg(test)]
+mod tests{
+    use bellman::Field;
+    use bellman::pairing::bn256::Bn256;
+    use franklin_crypto::alt_babyjubjub::{AltJubjubBn256, fs::Fs};
+    use franklin_crypto::eddsa::{PrivateKey, PublicKey, Signature, Seed};
+    use franklin_crypto::jubjub::{JubjubEngine,FixedGenerators};
+    use franklin_crypto::jubjub::edwards::Point;
+    use franklin_crypto::rescue::{RescueEngine, bn256::Bn256RescueParams};
+    use rand::{Rng, XorShiftRng, SeedableRng};
+    use crate::util::*;
+    use crate::verifier::MuSigVerifier;
+    use crate::errors::MusigError;
+    use crate::signer::MuSigSigner;
+
+
+    #[test]
+    fn test_musig_multiparty_full_round(){
+        let rng = &mut XorShiftRng::from_seed([0x3dbe6259, 0x8d313d76, 0x3237db17, 0xe5bc0654]);
+    
+        let jubjub_params = AltJubjubBn256::new();
+        let rescue_params = Bn256RescueParams::new_checked_2_into_1();
+        let generator = FixedGenerators::SpendingKeyGenerator;
+    
+        let message = random_message_hash(rng);
+    
+        let number_of_parties = 2;
```

Comment:
It seems that both tests are checking only 2 parties. Maybe it makes sense to add a test for a bigger amount of parties? 

And also, how the library will work if the number of parties equals to 1?